docs/site/BelongsTo-relation.md

```diff
@@ -75,18 +98,44 @@ export interface OrderRelations {
 export type OrderWithRelations = Order & OrderRelations;
 ```
 
-The definition of the `belongsTo` relation is inferred by using the `@belongsTo`
-decorator. The decorator takes in a function resolving the target model class
-constructor and designates the relation type. It also calls `property()` to
-ensure that the decorated property is correctly defined.
+The above example can also be expanded with explicit relation and property
+definitions as follows,
+
+- The standard naming convention for the foreign key property in the source
+  model is `target model name` + `Id`. (e.g. Order.customerId).
```

Comment:
In our documentation, we try to stay away from using `e.g.` since it doesn't translate well. It is best to use `for example`. See https://loopback.io/doc/en/contrib/Doc-style-guide.html#spelling-grammar-and-style

Reply:
agreed